deploy/olm-catalog/portworx/1.10.5/portworx-certified.clusterserviceversion.yaml

```diff
@@ -0,0 +1,540 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: portworx-operator.v1.10.5
+  namespace: placeholder
+  annotations:
+    capabilities: Auto Pilot
+    categories: "Storage"
+    description: Cloud native storage solution for production workloads
+    containerImage: portworx/px-operator:1.10.5-dev
+    repository: https://github.com/libopenstorage/operator
+    createdAt: 2023-02-22T17:24:13Z
+    support: Portworx, Inc
+    certified: "true"
+    operatorframework.io/initialization-resource: |-
+      {
+        "apiVersion": "core.libopenstorage.org/v1",
+        "kind": "StorageCluster",
+        "metadata": {
+          "name": "portworx",
+          "annotations": {
+            "portworx.io/is-openshift": "true"
+          }
+        }
+      }
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "core.libopenstorage.org/v1",
+          "kind": "StorageCluster",
+          "metadata": {
+            "name": "portworx",
+            "namespace": "test-operator",
+            "annotations": {
+              "portworx.io/is-openshift": "true"
+            }
+          },
+          "spec": {}
+        },
+        {
+          "apiVersion": "core.libopenstorage.org/v1",
+          "kind": "StorageNode",
+          "metadata": {
+            "name": "example",
+            "namespace": "test-operator"
+          },
+          "spec": {}
+        }
+      ]
+spec:
+  displayName: Portworx Enterprise
+  version: 1.10.5
+  minKubeVersion: "1.12.0"
+  maturity: stable
+  replaces: portworx-operator.v1.10.5
```

Comment:
1.10.4

Reply:
fixed. thanks for catching this, i will create ticket to automate this